lib/beaker/logger.rb

```diff
@@ -297,14 +305,18 @@ def success *args
     # @param args[Array<String>] Strings to be reported
     def notify *args
       return unless is_notify?
-      optionally_color BRIGHT_WHITE, *args
+      if @ci_build
```

Comment:
I liked that. I changed it a bit on the way up though. Because we only change two of the colors for CI output, I wanted that to stand out